onadata/apps/api/management/commands/delete_users.py

```diff
@@ -0,0 +1,32 @@
+from django.contrib.auth.models import User
+from django.core.management.base import BaseCommand
+
+
+class Command(BaseCommand):
+    help = 'Delete users'
+
+    def add_arguments(self, parser):
+        parser.add_argument('user', nargs='*')
+
+    def handle(self, *args, **kwargs):
+        users = kwargs['user']
+
+        if users:
+            for user in users:
+                user_details = user.split('-')
+                username = user_details[0]
+                email = user_details[1]
+                try:
+                    user = User.objects.get(username=username)
+                    user.delete()
```

Comment:
Can we have a confirmation that the person running the command can explicitly type yes to confirm they want to delete the account?